src/ExpressMiddlewares.js

```diff
@@ -15,21 +15,20 @@ class ExpressMiddlewares {
         const hostName = os.hostname();
         const serviceColor = process.env.SERVICE_COLOR || "unknown";
         const errorHandler = (request, response) => "error";
-        let optKeys = [];
-        let optKeysString = '';
+        const optKeys = [];
 
         // Check for additional access logs
-        if (opts && typeof opts === 'object') {
-          Object.keys(opts).forEach((key) => {
+        if (opts && typeof opts === "object") {
+          for (const key in opts) {
+            console.log(key);
             try {
-              morgan.token(key, typeof opts[key] === 'function' ? opts[key] : errorHandler);
+              morgan.token(key, typeof opts[key] === "function" ? opts[key] : errorHandler);
             }
             catch(err) {
               morgan.token(key, errorHandler);
             }
             optKeys.push(`\"${key}\": \":${key}\"`);
```

Comment:
might need a space here at the end `\"${key}\": \":${key}\" `

Reply:
I guess would be better to change delimiter to `", "` in this case, otherwise it will be like `"key": value ,"key": value` instead of `"key": value, "key": value`. But anyway, tell me what you think.